src/hooks/generate-development-docker-compose.ts

```diff
@@ -4,6 +4,7 @@ import { Inventory } from "../state/inventory.js";
 import { DockerComposeFileGenerator } from "../generator/docker-compose-file-generator.js";
 import { IConfig } from "@oclif/config";
 import loadConfig from "../helpers/config-loader.js";
+import { StateManager } from "../state/state-manager.js";
 
 // @ts-ignore
 export const hook: Hook<"generate-development-docker-compose"> = async function ({ serviceName, builderVersion, config }: { serviceName: string; builderVersion?: string; config: IConfig }) {
```

Comment:
I think I'd prefer it if this one had a flag in the arguments which decided whether to generate a spec with exclusions rather than having two hooks as they're very similar - they're only calling different methods